include/Grouper.h

```diff
@@ -40,26 +40,30 @@ struct groupStats{
   char country[3];
 };
 
-class Grouper {
- private:
-  sortField sorter;
-
-  int table_index;
+struct group {
   int64_t group_id_i;
```

Comment:
these group_id can be put together in a union as they're never used together. The only values set depends on the sorter.